domain-services/fiscal-event-service/pom.xml

```diff
@@ -76,26 +76,25 @@
       <artifactId>validation-api</artifactId>
     </dependency>
 
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-starter-data-mongodb</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>com.github.cloudyrock.mongock</groupId>
-      <artifactId>mongock-spring-v5</artifactId>
+   <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
     </dependency>
-
     <dependency>
-      <groupId>com.github.cloudyrock.mongock</groupId>
```

Comment:
Remove `mongock` related dependency also from `dependencyManagement` section.

Reply:
minor change, to be taken up as part of code cleanup.